docs/content/preview/explore/ysql-language-features/postgresql-compatibility.md

```diff
@@ -21,13 +21,28 @@ Because YugabyteDB is a distributed database, supporting all PostgreSQL features
 
 The following PostgreSQL features are not supported in YugabyteDB:
 
-- Pessimistic locking (except Read Committed, which is in beta, _supports_ [pessimistic locking](../../../architecture/transactions/read-committed/#cross-feature-interaction))
-- Table locks
-- [Inheritance](https://www.postgresql.org/docs/11/tutorial-inheritance.html)
-- Exclusion Constraints
-- GiST indexes
-- Kerberos/GSSAPI
-- Events (Listen/Notify)
-- Drop primary key
-- XML Functions
-- XA syntax
+| Unsupported PostgreSQL feature      | Track feature request GitHub issue |
+| ----------- | ----------- |
+| LOCK TABLE to obtain a table-level lock | [5384](https://github.com/yugabyte/yugabyte-db/issues/5384)|
+| Inheritance    | [5956](https://github.com/yugabyte/yugabyte-db/issues/5956)|
+| Exclusion constraints | [3944](https://github.com/yugabyte/yugabyte-db/issues/3944)|
+| Deferrable constraints | [1709](https://github.com/yugabyte/yugabyte-db/issues/1709)|
+| GiST indexes | [1337](https://github.com/yugabyte/yugabyte-db/issues/1337)|
+| Kerberos/GSSAPI | [7854](https://github.com/yugabyte/yugabyte-db/issues/7854)|
+| Events (Listen/Notify) | [1872](https://github.com/yugabyte/yugabyte-db/issues/1872)|
+| Drop primary key constraints |[8735](https://github.com/yugabyte/yugabyte-db/issues/8735)|
+| XML Functions | [1043](https://github.com/yugabyte/yugabyte-db/issues/1043)|
+| XA syntax | [11084](https://github.com/yugabyte/yugabyte-db/issues/11084)|
+| ALTER TYPE | [1893](https://github.com/yugabyte/yugabyte-db/issues/1893)|
+| CREATE CONVERSION | [10866](https://github.com/yugabyte/yugabyte-db/issues/10866)|
+| Primary/Foreign key constraints on foreign tables | [10698](https://github.com/yugabyte/yugabyte-db/issues/10698), [10699](https://github.com/yugabyte/yugabyte-db/issues/10699) |
+| LIKE ALL | [10697](https://github.com/yugabyte/yugabyte-db/issues/10697)|
+| GENERATED ALWAYS AS STORED columns | [10695](https://github.com/yugabyte/yugabyte-db/issues/10695)|
+| Multicolumn GIN indexes| [10652](https://github.com/yugabyte/yugabyte-db/issues/10652)|
+| CREATE ACCESS METHOD | [10693](https://github.com/yugabyte/yugabyte-db/issues/10693)|
+| ASC/DESC/HASH on GIN indexes | [10653](https://github.com/yugabyte/yugabyte-db/issues/10653)|
+| CREATE SCHEMA with elements | [10865](https://github.com/yugabyte/yugabyte-db/issues/10865)|
+| Index on citext column | [9698](https://github.com/yugabyte/yugabyte-db/issues/9698)|
+
```

Comment:
Do we want to have an exhaustive list of unsupported functionality? I think there are some more, some examples from a recent SQLsmith run:

> Type not yet supported in YugaByte: 702 (ABSTIME)
> Yugabyte does not support xid
> undefined datatype given to yb_hash_code
> gtsvector_in not implemented

Reply:
Hi @def- Do you have corresponding GH tickets for these unsupported features?

Reply:
No, just some I noticed in my testing. I didn't report them because the error messages sound like they are known not to be implemented, so they might not have a github issue.

Reply:
@def- could you please create the GitHub issues with the error messages you get and also commit to this PR with the additional rows in the table?

Reply:
Done. I think https://github.com/yugabyte/yugabyte-db/issues/15639 is probably rather yb-specific and https://github.com/yugabyte/yugabyte-db/issues/15640 also fails in postgres in the same way, so didn't add those two.

Reply:
Thanks @def-